.claude/skills/evaluation/references/nel-ci-guide.md

```
@@ -0,0 +1,275 @@
+MISTRAL3_PY=$(find /usr/local/lib -path "*/vllm/model_executor/models/mistral3.py" 2>/dev/null | head -1)
+sed -i "s/old_pattern/new_pattern/" "$MISTRAL3_PY"
+exec vllm serve /checkpoint --host 0.0.0.0 --port 8000 \
+    --tensor-parallel-size 4 --quantization modelopt_fp4 \
+    --trust-remote-code --served-model-name my-model --gpu-memory-utilization 0.9
+EOF
+chmod 777 /lustre/.../checkpoint/serve.sh'
+```
+
+**Step 2** — Set `NEL_DEPLOYMENT_COMMAND` to the wrapper:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)